app/Http/Controllers/API/v1/StatusController.php

```diff
@@ -315,9 +318,11 @@ public function destroy(int $id): JsonResponse {
      */
     public function update(Request $request, int $statusId): JsonResponse {
         $validator = Validator::make($request->all(), [
-            'body'       => ['nullable', 'max:280', 'nullable'],
-            'business'   => ['required', new Enum(Business::class)],
-            'visibility' => ['required', new Enum(StatusVisibility::class)],
+            'body'                      => ['nullable', 'max:280', 'nullable'],
+            'business'                  => ['required', new Enum(Business::class)],
+            'visibility'                => ['required', new Enum(StatusVisibility::class)],
+            'destinationId'             => ['required_with:destinationArrivalPlanned', 'exists:train_stations,id'],
+            'destinationArrivalPlanned' => ['required_with:destinationId', 'date'],
```

Comment:
Muss das in der OpenAPI im `components/schemas/StatusUpdateBody` abgebildet werden?

Reply:
Weiß ich nicht. Deswegen habe ich es auf das Issue von @HerrLevin geschrieben: #1043 (ganz unten)

Reply:
Ack.